src/defines.asm

```diff
@@ -218,10 +219,11 @@
 !sram_ctrl_kill_enemies = $702004
 !sram_ctrl_full_equipment = $702006
 !sram_ctrl_reset_segment_timer = $702008
-!sram_ctrl_load_state = $70200A
-!sram_ctrl_save_state = $70200C
-!sram_ctrl_load_last_preset = $70200E
-!sram_ctrl_random_preset = $702010
+!sram_ctrl_reset_segment_later = $70200A
```

Comment:
There should be init.asm changes to increment SRAM_VERSION and also initialize sram_ctrl_reset_segment_later to 0